engine/swarm/admin_guide.md

```diff
@@ -53,6 +53,29 @@ updates to the swarm. A quorum of managers must also agree on node additions
 and removals. Membership operations are subject to the same constraints as state
 replication.
 
+### Maintaining a quorum of managers
+
+Always maintain **an odd number of manager nodes** in the swarm. Otherwise, your
+swarm may lose a quorum of managers, and will not be able to perform management
+tasks. Three managers is the minimum, and five managers is recommended. In a
```

Comment:
One manager is the minimum. Three managers is the minimum for a setup to continue working after a manager failure. I don't know if we should go so far as to recommend 5 managers, as 3 is sufficient for a lot of use cases. But it is good to point out that 5 managers allows you to lose 2.